tests/playwright/sanity/container.test.ts

```diff
@@ -453,7 +453,7 @@ test.describe( 'Container tests @container', () => {
 		await editor.getPreviewFrame().locator( '.flex-preset-button' ).click();
 		await editor.getPreviewFrame().locator( '[data-preset="c100-c50-50"]' ).click();
 
-		await expect.soft( editor.getPreviewFrame().locator( '.e-con.e-con-full.e-con--column' ).last() ).toHaveCSS( 'padding', '0px' );
+		await expect.soft( editor.getPreviewFrame().locator( '.e-con.e-con-full.e-con--column' ).last() ).toHaveCSS( 'padding', '10px' );
```

Comment:
Please discuss this aspect with Doron and add it to the mind map as well. We need to be careful not to change the expected behavior.

Reply:
I traced and understood what was happening and fixed here